pandas/core/series.py

```diff
@@ -195,9 +196,12 @@ def __init__(self, data=None, index=None, dtype=None, name=None,
                 else:
                     data = data.reindex(index, copy=copy)
             elif isinstance(data, Categorical):
-                if dtype is not None:
+                # Allow dtype=category only, otherwise error
```

Comment:
yep - look over the test coverage for that and see if anything missing (out new tests near)